openwisp_monitoring/monitoring/models.py

```diff
@@ -9,13 +9,14 @@ class Meta(AbstractMetric.Meta):
         swappable = swappable_setting('monitoring', 'Metric')
 
 
-class Graph(AbstractGraph):
-    class Meta(AbstractGraph.Meta):
+class Chart(AbstractChart):
+    class Meta(AbstractChart.Meta):
         abstract = False
-        swappable = swappable_setting('monitoring', 'Graph')
+        swappable = swappable_setting('monitoring', 'Chart')
 
 
-class Threshold(AbstractThreshold):
-    class Meta(AbstractThreshold.Meta):
+class AlertThreshold(AbstractAlertThreshold):
+    class Meta(AbstractAlertThreshold.Meta):
         abstract = False
-        swappable = swappable_setting('monitoring', 'Threshold')
+        db_table = 'monitoring_alertthreshold'
```

Comment:
I prefer `monitoring_alert_threshold` for readability but not a big deal

Reply:
Django prefers to store and call it this way. For example the table names for `DeviceData` and `DeviceMonitoring` are `device_monitoring_devicedata` and `device_monitoring_devicemonitoring` respectively.

Reply:
why is db_table needed here?

Reply:
@nemesisdesign,  django's `migraions.RenameModel` works great for a simple rename but it needs you to manually create the migration script and take care of few things in case there is some alteration of fields too([docs](https://docs.djangoproject.com/en/3.0/ref/migration-operations/#renamemodel)).

Thus, there were two slight changes I had to make. One is the above one and the other one is contenttype registration. I hope that explains :smile: